src/components/FlowRunStateTypeTabs.vue

```diff
@@ -28,11 +30,23 @@
 
   const tabStates: Record<string, StateType[]> = {
     failed: ['failed', 'crashed'],
-    running: ['running', 'pending', 'paused'],
+    running: ['running', 'pending', 'cancelling'],
     completed: ['completed'],
-    scheduled: ['scheduled', 'cancelled'],
+    scheduled: ['scheduled', 'paused'],
+    cancelled: ['cancelled'],
   }
-  const tabs = Object.keys(tabStates)
+
+  const { count: cancelledCount } = useFlowRunsCount(getStateTypeFilter('cancelled'))
+
+  const tabs = computed(() => {
+    const tabNames = Object.keys(tabStates)
+
+    if (!cancelledCount.value || cancelledCount.value < 1) {
+      tabNames.splice(tabNames.indexOf('cancelled'), 1)
+    }
```

Comment:
nit: Filtering the array would be simpler 
```suggestion
    if (!cancelledCount.value || cancelledCount.value < 1) {
      tabNames.filter(value => value !== 'cancelled')
    }
```